### PR TITLE
Blog post 1.4.0

### PR DESCRIPTION
**Pull request by Zaxounette**

### CODE REVIEW

**Comment by Zaxounette** [comment]
Build fail is unrelated to the blog post itself:

`├─BROKEN─ http://halongmininginc.com/ (BLC_UNKNOWN)`

cc @dennisreimann

File: blog/_posts/btcpay-server-1-4-0.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: "BTCPay Server 1.4.0 - The Year of Change"
+date: 2022-01-27
```
Comment (Zaxounette) [inline blog/_posts/btcpay-server-1-4-0.md:3]: In the Netlify preview, it says "posted on the 28th", although I set the 27th right here.

possible bug?

**Review by Zaxounette — COMMENTED** [review]
.

File: blog/_posts/btcpay-server-1-4-0.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+category:
+  - "releases"
+tags:
+  - "btcpay-server"
+  - "release"
+  - "ui-ux"
+  - "greenfield"
+  - "api"
+  - "update"
+coverImage: "/images/BTCPayServer1.4.0.png"
```
Comment (pavlenex) [inline blog/_posts/btcpay-server-1-4-0.md:14]: Can we stop using these boring images?

**Comment by dstrukt** [comment]
Happy to help provide some assets for the blog / socials.

I remember the dimensions for the Open Graph Twitter, etc.. but would likely need them for the blog assets.

Just LMK, it's really easy for me :)

File: blog/_posts/btcpay-server-1-4-0.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+**BTCPay Server 1.4.0** is a major update that includes multiple items ranging from our massive UI revamp to updates to our Greenfield API. We won't cover them all in this release post, but as usual, the high-time preference crowd can view the [full Changelog](https://github.com/btcpayserver/btcpayserver/releases) directly.
+
+To update, simply navigate to your **Server Settings > Maintenance** tab and click **Update** or type *btcpay-update.sh* in the command line.
+
+## **UI overhaul** 🎨
+
+After six months of scoping, iterating and nearly two months of implementation tweaking and testing, we are finally ready to move to the next chapter of the BTCPay Server book.
+The goal behind these chages was to make the initial configuration as well as the first store creation and wallet setup as easy as possible for users who are new to BTCPay Server, while still enabling our power users to have easy access to the more complexe features available in BTCPay Server.
```
Comment (pavlenex) [inline blog/_posts/btcpay-server-1-4-0.md:27]:
```suggestion
The goal behind these changes was to make the initial onboarding easier. This includes, better guidance during the store creation and wallet setup as easy as possible to the new users, while still enabling our power users to have easy access to the more advanced features available.
```

File: blog/_posts/btcpay-server-1-4-0.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+**BTCPay Server 1.4.0** is a major update that includes multiple items ranging from our massive UI revamp to updates to our Greenfield API. We won't cover them all in this release post, but as usual, the high-time preference crowd can view the [full Changelog](https://github.com/btcpayserver/btcpayserver/releases) directly.
+
+To update, simply navigate to your **Server Settings > Maintenance** tab and click **Update** or type *btcpay-update.sh* in the command line.
+
+## **UI overhaul** 🎨
+
+After six months of scoping, iterating and nearly two months of implementation tweaking and testing, we are finally ready to move to the next chapter of the BTCPay Server book.
```
Comment (pavlenex) [inline blog/_posts/btcpay-server-1-4-0.md:26]:
```suggestion
After six months of scoping, iterating, and nearly two months of implementation, tweaking, and testing, we are finally ready to move to the next chapter of the BTCPay Server book.
```

File: blog/_posts/btcpay-server-1-4-0.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+----
+
+That's going to be all for highlights of this BTCPay Server 1.4.0 release! 
+It is now time for us to wish everyone the very best for the year 2022! 💚
+
+As usual, if you have problems, feedback, feature requests regarding BTCPay Server, feel free to reach out on our [community chat](https://chat.btcpayserver.org/). We hope you enjoy what this update has to offer.
+
+As always, thank you to our invaluable contributors, we love you:
+
+[@andhans](https://twitter.com/andhans_jail) [@britttttk](https://github.com/britttttk) [@dennisreimann](https://github.com/dennisreimann) [@dstrukt](https://github.com/dstrukt) [@Kukks](https://github.com/kukks/) [@NicolasDorier](https://github.com/nicolasdorier/) [@Pavlenex](https://github.com/pavlenex/) [@rockstardev](https://github.com/rockstardev/) [@ubolator](https://github.com/bolatovumar) [@woutersamaey](https://github.com/woutersamaey) [@Zaxounette](https://github.com/zaxounette)
```
Comment (pavlenex) [inline blog/_posts/btcpay-server-1-4-0.md:73]: Are you sure you've listed everyone who contributed to this release? I don't think andhans was involved in this release for example?

File: blog/_posts/btcpay-server-1-4-0.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+## **UI overhaul** 🎨
+
+After six months of scoping, iterating and nearly two months of implementation tweaking and testing, we are finally ready to move to the next chapter of the BTCPay Server book.
+The goal behind these chages was to make the initial configuration as well as the first store creation and wallet setup as easy as possible for users who are new to BTCPay Server, while still enabling our power users to have easy access to the more complexe features available in BTCPay Server.
+
+![BTCPay 1.4.0 Payment Methods view](/images/1.4.0_payment_methods.png)
+
+Without going into too much details (that you can find in our fully transparent [roadmap](https://github.com/orgs/btcpayserver/projects/19)), this overhaul is but a step into getting plugins as well as centralized store and instance dashboards into BTCPay Server.
+We hope you like these chages, and if you have any feedback feel free to reach out in our [community chat](https://chat.btcpayserver.org/), we'll be glad to have you.
+
```
Comment (pavlenex) [inline blog/_posts/btcpay-server-1-4-0.md:33]: @Zaxounette  We really should let people know about our new invoice statuses change (we have a warning on the invoice page)
- Paid = Settled 
- Processing = Settled 

We should highlight new features here:
- Login codes
- Two factor authentication with your Lightning wallet (LNURL-Auth)

Comment (pavlenex) [reply blog/_posts/btcpay-server-1-4-0.md:33]: Also we should talk more about plugins afterwards, afterall we are releasing LNBank, we have to mention it.

File: blog/_posts/btcpay-server-1-4-0.md (hunk 5 above)
Comment (Zaxounette) [reply blog/_posts/btcpay-server-1-4-0.md:73]:
```suggestion
[@britttttk](https://github.com/britttttk) [@dennisreimann](https://github.com/dennisreimann) [@dstrukt](https://github.com/dstrukt) [@Kukks](https://github.com/kukks/) [@NicolasDorier](https://github.com/nicolasdorier/) [@Pavlenex](https://github.com/pavlenex/) [@rockstardev](https://github.com/rockstardev/) [@ubolator](https://github.com/bolatovumar) [@woutersamaey](https://github.com/woutersamaey) [@Zaxounette](https://github.com/zaxounette)
```

I checked against the names in the Changelog. Forgot to take out Andhans.
Left in Core contributors like Rockstar and Britt, even though they don't appear in the Changelog.

File: blog/_posts/btcpay-server-1-4-0.md (hunk 6 above)
Comment (Zaxounette) [reply blog/_posts/btcpay-server-1-4-0.md:33]: Regarding Plugins, noted.

Regarding invoice status change, I mention it in "Misc".

Comment (pavlenex) [reply blog/_posts/btcpay-server-1-4-0.md:33]: Imo invoice statuses should not be in misc, it's a big change and we should emphasize it as people's processes depend on it.

**Comment by NicolasDorier** [comment]
I think we can leave out the changelog and just have the link to github.

**Pull request merged**